src/app/mod.rs

```diff
@@ -475,6 +497,8 @@ impl App {
             BookmarkDel => self.delete_bookmark(),
             BookmarkEditTags => self.edit_bookmark_tags(),
             ClearCache => self.clear_cache(),
+            ClickNextPage => self.server_send(InnerMessage::ClickNextPage()),
```

Comment:
I used to put the calls to `server_send()` in `src/app/server.rs` but that might not be needed anymore since most of them are one-liner.
Which way do you think is best?
I see two avantages to stick with the old way:

 * We don't need to fully qualify the message in case it's the same as a command.
 * This let this bug `handle_command()` method cleaner.